src/odb.h

```diff
@@ -112,6 +112,12 @@ int git_odb__error_mismatch(
 int git_odb__error_notfound(
 	const char *message, const git_oid *oid, size_t oid_len);
 
+/*
+ * Generate an error for null OID and return the specified error
+ * code.
+ */
+int git_odb__error_null_oid(int error, const char *message);
```

Comment:
Do we need to have this in the header?  Why not just make `git_odb__error_null_oid` static in `odb.c`?

Reply:
No, definitly not. I think I just blindly followed style of the other error functions without thinking